packages/core/android/build.gradle

```diff
@@ -41,5 +41,5 @@ dependencies {
     api 'com.segment.analytics.android:analytics:4.+'
 
     api 'com.facebook.react:react-native:+'
-    api 'org.jetbrains.kotlin:kotlin-stdlib:1.2.+'
+    api 'org.jetbrains.kotlin:kotlin-stdlib:1.3.+'
```

Comment:
should we use a specific version rather than latest for these?

Reply:
I think it originally was set by IDEA, do you think it'd be better to not specify any?

Reply:
I think we should be more specific (e.g. 1.3.2), rather than loose. Otherwise, it might work now but fail later.